DTDL/tcads_source.json

```diff
@@ -0,0 +1,41 @@
+{
+    "@context" : "dtmi:dtdl:context;2",
+    "@id" : "dtmi:io:tributech:source:tcads;1", 
+    "@type": "Interface", 
+    "@extends": "dtmi:io:tributech:source:base;1",
+    "displayName": "Beckhoff TwinCat ADS Source", 
+    "contents" : [
+        {
+            "@type" : "Property", 
+            "name" : "ConnectionString", 
```

Comment:
Should we maybe call it `NetId` since it seems more common the Beckhoff world?
https://infosys.beckhoff.com/content/1033/tc3_adsnetref/7313101963.html?id=1308284143287996872

Reply:
I thought we should use the same Term in our Enviroment and use the DisplayName Tag for the Explanation to the User ? But i will change it.

Reply:
Of course there is the more important property description for the consumers but even in our environment you used `AmsNetId` so I considered unifying it.